docs/clustertemplate.md

```diff
@@ -1,36 +1,24 @@
-# Next-gen Cluster Provisioning using Cluster templates
-
-This document describes a new way of provisioning workload clusters in Arlon.
-The most significant change is the *Cluster Template* construct, which replaces the older ClusterSpec from gen1 clusters.
-To distinguish them from the older gen1 clusters, the ones deployed from a cluster template are called next-gen clusters or gen2 clusters.
-
-## Goals
+# Cluster Templates
 
+The Cluster Template object is designed with the following goals:
 - Allow users to deploy arbitrarily complex clusters using the full Cluster API feature set.
 - Fully declarative and gitops compatible: a cluster deployment should be composed of one or more self-sufficient manifests that the user can choose to either apply directly (via kubectl) or store in git for later-stage deployment by a gitops tool (mainly ArgoCD).
```

Comment:
A clustertemplate does involve a "prep" step.done by the arlon CLI, so should we call the clustertemplates fully declarative?

Reply:
Good point. I think we still should, but we should discuss this topic